common/djangoapps/entitlements/models.py

```diff
@@ -24,3 +59,5 @@ class CourseEntitlement(TimeStampedModel):
         help_text='The current Course enrollment for this entitlement. If NULL the Learner has not enrolled.'
     )
     order_number = models.CharField(max_length=128, null=True)
+    policy = models.ForeignKey(CourseEntitlementPolicy, null=True)
+    objects = CourseEntitlementManager()
```

Comment:
If we use this manager for CourseEntitlements, will we have to scan/update the whole table anytime we want to query for a `CourseEntitlement`?

Reply:
Wrote this above but just going through and answering the whole list.

>Yes, looking into creating a property now that we'll need to use over the default expired_at when using the ORM, and will be overriding the GET and LIST methods for the API views. This'll keep the subset we iterate on and updated down to a minimum.